docs/GetStarted/getting_started_macos_bazel.md

```diff
@@ -0,0 +1,164 @@
+# Getting Started on MacOS with Bazel
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+
+    * This document parallels getting_started_linux_bazel.md and
+      getting_started_windows_bazel.md
+      Please keep them in sync.
+-->
+
+This guide walks through building the core compiler and runtime parts of IREE
+from source. Auxiliary components like the Python bindings and Vulkan driver are
+not documented for MacOS at this time.
+
+IREE is not officially supported on MacOS at this time. It may work, but it is
+not a part of our open source CI, and may be intermittently broken.
+Contributions related to MacOS support and documentation are welcome however.
+
+## Prerequisites
+
+### Install Bazel
+
+Install the version of Bazel found in
+[`.bazelversion`](https://github.com/google/iree/blob/master/.bazelversion).
```

Comment:
Link to docs (https://docs.bazel.build/versions/master/install.html or https://docs.bazel.build/versions/master/install-os-x.html). Can copy language from the other getting_started_[os]_bazel.md docs.

Reply:
Link to the docs for installing from a binary are on on line 38. Explanation of why I linked them there in the convo below.